adafruit_seesaw/seesaw.py

```diff
@@ -291,7 +291,7 @@ def analog_write(self, pin, value):
         if self.pin_mapping.pwm_width == 16:
             if pin in self.pin_mapping.pwm_pins:
                 pin_found = True
-                cmd = bytearray([self.pin_mapping.pwm_pins.index(pin), (value >> 8), value])
```

Comment:
I'd recommend using `struct.pack` to do the byte packing instead. It'll be fewer allocations because it'll produce a byte string. As written, this line needs to construct a temporary list and then a bytearray.

Reply:
@tannewt ill put an issue, i want to get this working as-is first :)